tests/pr_test.go

```diff
@@ -29,6 +29,8 @@ const add_atracker_route = false
 var ignoreUpdates = []string{
 	"module.landing_zone.module.landing_zone.ibm_atracker_target.atracker_target[0]",
 	"module.landing_zone.ibm_atracker_target.atracker_target[0]",
+	"module.landing_zone.module.vpc[\"management\"].ibm_is_network_acl.network_acl[\"management-acl\"]",
```

Comment:
Why are ignoreUpdates being added to the pr_test for network acl? Was it to prevent the upgrade test from failing?

Reply:
I think the answer is that we are expecting ACLs rules to be deleted from the ACL resource. But not the resource itself, so they come in as updates that we need to ignore (for this PR only)

Reply:
Resolving this, please ignore, was explained that this exception will be removed immediately after patch is released.